pkgs/by-name/pc/pcsx2/package.nix

```
@@ -144,9 +16,23 @@ llvmPackages_17.stdenv.mkDerivation (finalAttrs: {
       PC, with many additional features and benefits.
     '';
     homepage = "https://pcsx2.net";
-    license = with licenses; [ gpl3Plus lgpl3Plus ];
-    maintainers = with maintainers; [ hrdinka govanify ];
+    license = with licenses; [
+      gpl3Plus
+      lgpl3Plus
+    ];
+    maintainers = with maintainers; [
+      hrdinka
+      govanify
+      matteopacini
+    ];
     mainProgram = "pcsx2-qt";
-    platforms = [ "x86_64-linux" ];
+    platforms = [ "x86_64-linux" ] ++ platforms.darwin;
```

Given the latest comments as this PR stands now I kindly request the following change:
```suggestion
    platforms = [ "x86_64-linux" "x86_64-darwin" ];
```